code/go/internal/validator/semantic/validate_stream_input_materialized.go

```
@@ -0,0 +1,160 @@
+	return errs
+}
+
+// validatePolicyTemplateInputsMaterialized checks the package-level manifest.yml for
+// policy_template inputs that carry a source-only 'package:' field instead of 'type:'.
+func validatePolicyTemplateInputsMaterialized(fsys fspath.FS) specerrors.ValidationErrors {
+	manifestRelPath := "manifest.yml"
+	data, err := fs.ReadFile(fsys, manifestRelPath)
+	if err != nil {
+		// If there is no manifest we have nothing to check; other validators handle
+		// the missing-manifest case.
+		return nil
```

golangci-lint (2.12.2)

[error] error is not nil (line 125) but it returns nil

(nilerr)